Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM node:14
+
+# Create app directory
+WORKDIR /usr/src/app
+
+COPY package*.json ./
+
+RUN npm install
+
+COPY . .
+
+EXPOSE 3000
```

Comment:
open port 3000 on image this lines up with port that the application listens on. Allowing the app to be hit by the outside world